go-controller/pkg/clustermanager/zone_cluster_controller.go

```diff
@@ -165,6 +189,30 @@ func (zcc *zoneClusterController) handleAddUpdateNodeEvent(node *corev1.Node) er
 			node.Name, err)
 	}
 
+	if config.OVNKubernetesFeature.EnableInterconnect {
```

Comment:
We got to do something for these crazy people :)

I guess there is no harm in having these annotations.  We could definitely clear the annotation if present.

This may complicate the code a bit.  I'll see if that is straightforward.  Otherwise will the TODO comment to cleanup the annotation would do ?

Reply:
I've added TODO